### ISSUE TITLE
Angular 13 Support

### ISSUE THREAD

**Issue by onclave** [peterfreeman/ngx-dropzone#137]
After upgrading to angular 13, this warning is thrown during the build process:

```
⠙ Generating browser application bundles (phase: setup)...Processing legacy "View Engine" libraries:
- ngx-dropzone [es2015/esm2015] (git+https://github.com/peterfreeman/ngx-dropzone.git)
Encourage the library authors to publish an Ivy distribution.
```

Despite the warning, everything works fine, though.

Thank you for this awesome library, by the way!

**Comment by peterfreeman** [comment]
Hi @onclave,

thank you for the hint. I am currently working on the next major update which will support Angular 13 by default.
I plan to release it in the next few weeks. If anyone of you has time to fix this warning (should be config only), please open a PR.

Thank you and happy coding! 🤓

**Comment by djbreen7** [comment]
Any updates on this?

**Comment by peterfreeman** [comment]
Merged #139 and released updated package v3.1.0.
